js/src/components/paid-ads/continue-button.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppButton from '.~/components/app-button';
+
+/**
+ * Renders Continue button on paid ad campaign create and edit page.
+ *
+ * @param {Object} props Props
+ * @param {Object} props.formProps Form props forwarded from `Form` component.
+ * @param {Function} props.handleContinueClick Function to handle the continue button click.
+ * @return {import(".~/components/paid-ads/continue-button").default} The button.
+ */
+const ContinueButton = ( { formProps, handleContinueClick } ) => {
```

Comment:
The props naming convention of an event handler usually starts with `on` by the name of the event. A name starting with 'handle' is used for an event handler set to the corresponding prop starting with `on`.

The `handleContinueClick` is directly forwarded to the button, so the naming would be better to use `onClick`.

Reply:
Understood. Changed the name to onClick.